tools/developer.html

```diff
@@ -70,7 +70,7 @@ <h3>Code</h3>
     </div>
     <div>
       <h3>Rendered in <span class="rendering_time">...</span> ms [<span class="rendering_stats">...</span>]</h3>
-      <pre><code class="hljs">...</code></pre>
+      <highlightjs auto :code="code" ignore_illegals />
```

Comment:
I see `ignore_illegals` was commented out in the component, should it be removed from here too?